lib/danger/scm_source/git_repo.rb

```diff
@@ -53,7 +53,7 @@ def exec(string)
       Dir.chdir(self.folder || ".") do
         git_command = string.split(" ").dup.unshift("git")
         Open3.popen2(default_env, *git_command) do |_stdin, stdout, _wait_thr|
-          stdout.read.rstrip
+          stdout.read.scrub.rstrip
```

Comment:
@fuzzyrichie Sorry I don't understand what *invalid UTF-8 character* comes here.

Could you describe minimal steps to reproduce it, and add some test cases?

Reply:
I'll look to get a sample of the output that caused the error, give me a day or two to set that up!